src/components/containers/MenuPanel.js

```diff
@@ -32,7 +38,7 @@ export default class MenuPanel extends Component {
 
   render() {
     const {show, ownline, label, children} = this.props;
-    const isOpen = show || this.state.isOpen;
+    const {isOpen = show} = this.state;
```

Comment:
@VeraZab @aulneau the logic here has changed. Before we would take `show` first and then `isOpen` after. Now `state.isOpen` takes precedence. 

What that means is that the `show` props is effectively useless because `this.state.isOpen` is always defined.

This is the source of the test failure

Reply:
Whoops you're totally right! that's my bad, I changed that line. Thanks @bpostlethwaite

Reply:
The previous logic `show || this.state.isOpen;` is accurate. If `show` is `false` then defer to the component state. If show is `true` then the component will always be open.